src/sass/core/_main.scss

```diff
@@ -419,7 +419,6 @@ legend {
     }
 
     input[type=text],
-    .print-input-text,
```

Comment:
It is important to use the same style as for input fields. Not doing this re-creates a bunch of issues that were fixed by  08238fcf242bb0072447945a647535218fa46fdd (and a few later commits)

Reply:
got it, undo the CSS changes